services/web-app/pages/designing/get-started.mdx

```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
```

Comment:
Add wireframe design kits to Sketch tab (they are in the schema so hopefully that isn't too hard to add)

Reply:
@aubrey-oneal adding this to new issue